SAP2000_Adapter/CRUD/Create/Bar.cs

```diff
@@ -145,6 +152,114 @@ private bool SetObject(Bar bhBar)
+            }
+
+            // Design Procedure
+
+            BarDesignProcedure barDesignProcedure = bhBar.DesignProcedure();
+
+            if (barDesignProcedure != null)
+            {
+                // issue with cold form as a material not being able to be pushed...
+                if (barDesignProcedure.DesignProcedure == DesignProcedureType.Aluminum ||
+                    barDesignProcedure.DesignProcedure == DesignProcedureType.ColdFormed ||
+                    barDesignProcedure.DesignProcedure == DesignProcedureType.Steel ||
+                    barDesignProcedure.DesignProcedure == DesignProcedureType.Concrete)
+                {
+                    // Design Procedure "MyType" is 1 if specified from material list available (limited to enums shown)
+                    if (m_model.FrameObj.SetDesignProcedure(name, 1, 0) != 0)
+                    {
+                        CreatePropertyWarning("DesignProcedure", "Bar", name);
+                    }
+                    else
+                    {
+                        Engine.Reflection.Compute.RecordNote($"Bar {bhBar.Name} with SAP id {name} was set with a design procedure automatically based on its material (Steel/Concrete/Cold Form/Aluminum) regardless of selected input.");
+                    }
+                }
+                else
+                {
+                    // Design Procedure "MyType" is 2 if no design specified - this defaults to aluminum rather than nodesign in the api call...
+                    if (m_model.FrameObj.SetDesignProcedure(name, 2, 0) != 0)
+                    {
+                        CreatePropertyWarning("DesignProcedure", "Bar", name);
+                    }
+                    else
+                    {
+                        Engine.Reflection.Compute.RecordNote($"Bar {bhBar.Name} with SAP id {name} does not have a design procedure.");
+                    }
+
+                }
+            }
+
+            // Insertion Point Offset
+
+            Offset offset = bhBar.Offset;
+
+            double[] offset1 = new double[3];
+            double[] offset2 = new double[3];
+
+            if (offset != null)
+            {
+                if (offset.Start != null)
+                {
+                    offset1[1] = offset.Start.Z;
+                    offset1[2] = offset.Start.Y;
+                }
+
+                if (offset.End != null)
+                {
+                    offset2[1] = offset.End.Z;
+                    offset2[2] = offset.End.Y;
+                }
+            }
+
+            if (m_model.FrameObj.SetInsertionPoint(name, (int)bhBar.InsertionPoint(), false, bhBar.ModifyStiffnessInsertionPoint(), ref offset1, ref offset2) != 0)
+            {
+                CreatePropertyWarning("Insertion point and perpendicular offset", "Bar", name);
+            }
+            
+            // Section Property Modifiers
+
+            if (bhBar.SectionProperty.FindFragment<SectionModifier>() != null)
```

Comment:
same as below, this should be looking for modifiers on the bar, not on the section property

Reply:
Per discussion, removed section property modifiers till later PR.